financial-connections/src/main/java/com/stripe/android/financialconnections/features/common/LoadingContent.kt

```diff
@@ -137,7 +160,84 @@ internal fun LoadingSpinner(
     }
 }
 
-private const val LOADING_SPINNER_ROTATION_MS = 1000
+@Composable
+internal fun LoadingPill(
```

Comment:
maybe `LoadingPillContainer` or something similar to differenciate it with the actual Pill view?